packages/clerk-js/src/core/resources/Session.ts

```diff
@@ -267,9 +268,24 @@ export class Session extends BaseResource implements SessionResource {
       return cachedToken.getRawString() || null;
     }
     const path = template ? `${this.path()}/tokens/${template}` : `${this.path()}/tokens`;
+
     // TODO: update template endpoint to accept organizationId
-    const params = template ? {} : { organizationId };
-    const tokenResolver = Token.create(path, params);
+    const params: Record<string, string | null> = template ? {} : { organizationId };
+
+    const createTokenWithCaptchaProtection = async () => {
+      try {
+        return Token.create(path, params);
+      } catch (e) {
+        if (isClerkAPIResponseError(e) && e.errors[0].code === 'requires_captcha') {
```

Comment:
Can we extract the `requires_captcha` error code to a variable. I can see it is used twice!

Reply:
I'm going to refactor this logic out once we've verified that bot protection works as expected